Workbooks/Workloads-AMA/SQL Server Insights/SQL Server Insights.workbook

```diff
@@ -181,7 +181,7 @@
 to [this document](https://docs.microsoft.com/en-us/sql/relational-databases/system-dynamic-management-views/sys-dm-db-resource-stats-azure-sql-database?view=azuresqldb-current).\",\r\n        \"author\": \"Microsoft\",\r\n        \"uniqueTag\": \"sql-data-io-utilization\",\r\n        \"recommendationLevel\": 4,\r\n\t\t\"severity\": \"4\",\r\n        \"applyTo\": [],\r\n        \"alertTemplate\": {\r\n            \"type\": \"log-metric-measurement\",\r\n            \"query\": \"InsightsMetrics | where Namespace == 'sqlserver_azure_db_resource_stats' and Name == 'avg_data_io_percent' | extend Tags = todynamic(Tags) | where tostring(Tags.['azm.ms/profileid']) == '{ProfileId}' | extend Instance = Tags.sql_instance, Database = Tags.database_name | summarize AggregatedValue = percentile(Val,95) by bin(TimeGenerated, 5m), tostring(Instance), tostring(Database) //| where AggregatedValue > {AlertThreshold}\",\r\n            \"thresholdOperator\": \"GreaterThan\",\r\n            \"defaultThreshold\": \"95\",\r\n            \"metricMeasurement\": {\r\n                \"thresholdOperator\": \"GreaterThan\",\r\n                \"threshold\": \"3\",\r\n                \"metricTriggerType\": \"Consecutive\",\r\n                \"metricColumn\": \"Instance,Database\"\r\n            }\r\n        }\r\n    },\r\n    {\r\n        \"name\": \"Azure DB Log write utilization\",\r\n        \"description\": \"Fired when Azure DB Log write utilization percentage exceeds the specified threshold. Looks at the 95th percentile value over 5 minutes, and if it exceeds the threshold over 3 consecutive time periods the alert is triggered. For more details on this metric, refer to [this document](https://docs.microsoft.com/en-us/sql/relational-databases/system-dynamic-management-views/sys-dm-db-resource-stats-azure-sql-database?view=azuresqldb-current).\",\r\n        \"author\": \"Microsoft\",\r\n        \"uniqueTag\": \"sql-log-write-utilization\",\r\n        \"recommendationLevel\": 4,\r\n\t\t\"severity\": \"4\",\r\n        \"applyTo\": [],\r\n        \"alertTemplate\": {\r\n            \"type\": \"log-metric-measurement\",\r\n            \"query\": \"InsightsMetrics | where Namespace == 'sqlserver_azure_db_resource_stats' and Name == 'avg_log_write_percent' | extend Tags = todynamic(Tags) | where tostring(Tags.['azm.ms/profileid']) == '{ProfileId}' | extend Instance = Tags.sql_instance, Database = Tags.database_name | summarize AggregatedValue = percentile(Val,95) by bin(TimeGenerated, 5m), tostring(Instance), tostring(Database) //| where AggregatedValue > {AlertThreshold}\",\r\n            \"thresholdOperator\": \"GreaterThan\",\r\n            \"defaultThreshold\": \"95\",\r\n            \"metricMeasurement\": {\r\n                \"thresholdOperator\": \"GreaterThan\",\r\n                \"threshold\": \"3\",\r\n                \"metricTriggerType\": \"Consecutive\",\r\n                \"metricColumn\": \"Instance,Database\"\r\n            }\r\n        }\r\n    }\r\n]\r\n",
```

Comment:
1. "name\": \"Azure DB, MI and SQL Server Log file utilization\"
In the query it says `where Database_Type != 'AzureSQLDB'`. Should the name omit `Azure MI and SQL Server Log utilization`
2. I see some instances of `//| where AggregatedValue > {AlertThreshold}` does this mean this is commnted out? if so do we need this?

Reply:
yeah, #2 is comment that we want to keep as reference as the threshold we recommended